src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -1003,4 +1013,8 @@
   </xsl:attribute-set>
   -->
 
+  <!-- Provide enough space for two digit labels (needed in bibliography). -->
```

Comment:
The only problem here is that that numbers of left aligned instead of right aligned. I'm not sure if that's possible to configure, but it would make it look a lot nicer.